onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3425,10 +3425,58 @@ def aten_meshgrid(tensors: Sequence[TensorType]) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_min(self: TensorType) -> TensorType:
+@torch_op("aten::max", trace_only=True)
+def aten_min(
+    self: TReal, dim_or_other: Union[TReal, INT64] = None, keepdim: BOOL = None
+) -> TReal:
     """min(Tensor self) -> Tensor"""
```

Comment:
Is this signature accurate from native_functions? Should we create overloads for different aten overloads?

Reply:
yes, we have 3 overload

Reply:
I meant public overloads

Reply:
sorry, not got your idea ~